From 0325b986cc47d8ea4d68954138b846bd6ef4d530 Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@freescale.com>
Date: Thu, 21 Nov 2013 20:43:13 -0200
Subject: [PATCH 86/95] ARM: dts: imx6sabre: Pass reg property in lvds

The 'reg' property is missing in the lvds nodes, so add them for correctness.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx6q-sabrelite.dts  | 1 +
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6q-sabrelite.dts b/arch/arm/boot/dts/imx6q-sabrelite.dts
index fca8f220..0f003d0 100644
--- a/arch/arm/boot/dts/imx6q-sabrelite.dts
+++ b/arch/arm/boot/dts/imx6q-sabrelite.dts
@@ -169,6 +169,7 @@
 	status = "okay";
 
 	lvds-channel@0 {
+		reg = <0>;
 		fsl,data-mapping = "spwg";
 		fsl,data-width = <18>;
 		status = "okay";
diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 570767f..381c499 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -236,6 +236,7 @@
 	status = "okay";
 
 	lvds-channel@1 {
+		reg = <1>;
 		fsl,data-mapping = "spwg";
 		fsl,data-width = <18>;
 		status = "okay";
-- 
1.8.4.3

